webui/react/config-overrides.js

```diff
@@ -23,7 +23,7 @@ module.exports = override(
   }),
 
   // Add LESS loader support for antd.
-  addLessLoader({ javascriptEnabled: true }),
+  addLessLoader({ lessOptions: { javascriptEnabled: true } }),
```

Comment:
**question:** did I just set this wrong or was this a change due to an upgraded `customize-cra` version?

Reply:
less loader upgrade now needs less options to be inside an object . one of the breaking changes.